### PR TITLE
Update the accessibility statement page

### PR DESCRIPTION
**Pull request by Windvis**
With the date picker issue being resolved in #88 the website should be fully accessible.

(Large diff because I formatted with Prettier, which we should set up for .hbs files anyways).

### CODE REVIEW

File: app/templates/accessibility-statement.hbs [hunk 1]
```diff
@@ -5,40 +5,65 @@
...
-        <h2>Niet-toegankelijke inhoud</h2>
-
-        <p>De onderstaande inhoud is niet-toegankelijk om de volgende reden(en):</p>
-        <ul>
-          <li>
-            <strong>Datum selectie:</strong> het datum selectie veld bevat een datum popup die niet toegankelijk is via het toetsenbord. (<AuLinkExternal href="https://www.w3.org/Translations/WCAG21-nl/#toetsenbord">Succescriterium 2.1.1 Toetsenbord</AuLinkExternal>). <br>
-            <em>Alternatief:</em> een datum kan via tekst ingegeven worden.
-            <br>
-          </li>
-        </ul>
```
Comment (Windvis) [inline app/templates/accessibility-statement.hbs:23]: This section was removed.

File: app/templates/accessibility-statement.hbs [hunk 2]
```diff
@@ -5,40 +5,65 @@
...
+            href="http://www.ejustice.just.fgov.be/eli/decreet/2018/12/07/2018032457/justel"
+          >Bestuursdecreet van 7 december 2018</AuLinkExternal>.
+        </p>
+        <p>
+          Deze toegankelijkheidsverklaring is van toepassing op
+          <AuLinkExternal href="https://...">....vlaanderen.be</AuLinkExternal>
+        </p>
 
         <h2>Nalevingsstatus</h2>
-        <p>Deze web applicatie voldoet gedeeltelijk aan de <AuLinkExternal href="https://www.w3.org/Translations/WCAG21-nl/">Richtlijnen voor Toegankelijkheid van Webcontent (WCAG) 2.1 Niveau AA</AuLinkExternal>, omdat niet aan de onderstaande eis(en) is voldaan.</p>
```
Comment (Windvis) [inline app/templates/accessibility-statement.hbs:12]: This was slightly modified.

File: app/templates/accessibility-statement.hbs [hunk 3]
```diff
@@ -5,40 +5,65 @@
...
 
         <h2>Feedback en contactgegevens</h2>
-        <p>Voor alle vragen en opmerkingen over de toegankelijkheid van de vermelde websites van het Agentschap Binnenlands Bestuur kunt u contact opnemen via <a href="mailto:binnenland@vlaanderen.be" class="c-link">binnenland@vlaanderen.be</a></p>
+        <p>
+          Voor alle vragen en opmerkingen over de toegankelijkheid van de
+          vermelde websites van het Agentschap Binnenlands Bestuur kunt u
+          contact opnemen via
+          <a
+            href="mailto:binnenland@vlaanderen.be"
+            class="au-c-link"
```
Comment (Windvis) [inline app/templates/accessibility-statement.hbs:43]: Fixed some link styling, `c-link` doesn't exist.

**Review by aliokan — APPROVED** [review]
Looks good, thanks!

**Pull request merged**